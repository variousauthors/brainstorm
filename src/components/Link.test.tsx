import React from 'react';
import renderer from 'react-test-renderer';
import Link from './Link';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <Link page="http://www.facebook.com">Facebook</Link>,
  );
  let tree: any = component.toJSON();

  expect(tree).not.toBe(null);

  if (tree === null) fail()

  expect(tree).toMatchSnapshot();
});
