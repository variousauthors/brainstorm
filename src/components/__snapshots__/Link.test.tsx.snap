// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<a
  className="normal"
  href="http://www.facebook.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebook
</a>
`;
